src/lib.rs

```diff
@@ -947,7 +946,6 @@ impl Drop for Rav1dContext {
             let _ = mem::take(&mut self.tiles);
             let mut n_4 = 0;
             while n_4 < 8 {
-                rav1d_cdf_thread_unref(&mut *(self.cdf).as_mut_ptr().offset(n_4 as isize));
```

Comment:
Do we not need to set the `Option`s in `self.cdf` to `None` here the way we do for the other calls to `rav1d_cdf_thread_unref`?

Reply:
Addressing in other thread